docs/src/emulation.md

```diff
@@ -408,10 +556,6 @@ await using var context = await browser.NewContextAsync(new()
 
 Change the location later:
 
-```js
```

Comment:
I think it was good like that. Otherwise, we lack the snippet for js. I'd only replace `context` with `page.context()` to start with the page that is in all the snippets.

Reply:
your right. moved it back